pkg/extender/extender.go

```diff
@@ -228,6 +230,15 @@ func (e *Extender) processFilterRequest(w http.ResponseWriter, req *http.Request
 			storklog.PodLog(pod).Errorf("Error getting list of driver nodes, returning all nodes, err: %v", err)
 		} else {
 			for _, volumeInfo := range driverVolumes {
+				// Pod is using a volume that is labeled for Windows
```

Comment:
Since `windows` is specific to Portworx driver, can we move this logic into the Portworx driver? 
Add a new field in the Volume object as `WindowsShare` and then add a check here something like
```
if volumeInfo.WindowsVolume {
  e.encodeFilterResponse()
}
```

Reply:
Done